testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/cluster/JGroupsCertificateRotationClusterTest.java

```diff
@@ -21,91 +21,134 @@ public class JGroupsCertificateRotationClusterTest extends AbstractClusterTest {
     @Test
     public void testRotation() {
         Assume.assumeTrue(getClusterSize() >= 2);
-        var mtlsEnabled = assumeEnabledAndOverwriteRotation(1, TimeUnit.DAYS);
-        Assume.assumeTrue(mtlsEnabled);
-        assertClusterSize();
+        Assume.assumeTrue(isMtlsEnabled());
 
-        var alias = currentCertificateAliasFor(0);
-        log.infof("Current JGroups Certificate alias: %s", alias);
+        try (var revert = overwriteRotation(1, TimeUnit.DAYS)) {
```

Comment:
using the auto-closable approach, you don't need to overwrite the rotation in this test.